test/fixtures/users.yml

```diff
@@ -7,3 +7,10 @@
 # two:
 #   name: MyString
 #   email: MyString
+
+#note to use spaces and not tabs for indentations in yml files
+michael:
+ name: Michael Example
+ email: michael@example.com
+ password_digest: <%= User.digest('password') %>
+ 
```

Comment:
When creating a new line at the end of the file make sure it has no spaces but just the new line
```suggestion

```